subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -36,11 +36,19 @@
 #define PDU_AC_PAYLOAD_SIZE_MIN 1
 /* Advertisement channel maximum legacy payload size */
 #define PDU_AC_LEG_PAYLOAD_SIZE_MAX 37
+/* Advertisement channel maximum legacy advertising/scan data size */
+#define PDU_AC_LEG_DATA_SIZE_MAX   31
 /* Advertisement channel maximum extended payload size */
 #define PDU_AC_EXT_PAYLOAD_SIZE_MAX 255
 
 /* Advertisement channel maximum payload size */
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
+#if defined(CONFIG_BT_BROADCASTER)
+#define BT_CTLR_ADV_DATA_LEN_MAX CONFIG_BT_CTLR_ADV_DATA_LEN_MAX
+#else /* !CONFIG_BT_BROADCASTER */
+#define BT_CTLR_ADV_DATA_LEN_MAX PDU_AC_LEG_DATA_SIZE_MAX
```

Comment:
Why do we need to define `BT_CTLR_ADV_DATA_LEN_MAX` if `CONFIG_BT_BROADCASTER` is not enabled?